libs/coin-framework/src/api/types.ts

```diff
@@ -39,7 +43,7 @@ export type Api = {
   combine: (tx: string, signature: string, pubkey?: string) => string;
   craftTransaction: (address: string, transaction: Transaction, pubkey?: string) => Promise<string>;
   estimateFees: (addr: string, amount: bigint) => Promise<bigint>;
-  getBalance: (address: string) => Promise<bigint>;
+  getBalance: (address: string) => Promise<Asset | bigint>;
```

Comment:
Ok for now but I guess we want to update all coin modules with only asset tree or array as output ? (As we discussed with cloud wallet)